addons/account_edi/data/cron.xml

```diff
@@ -3,7 +3,7 @@
         <field name="name">EDI : Perform web services operations</field>
         <field name="model_id" ref="model_account_edi_document"/>
         <field name="state">code</field>
-        <field name="code">model.search([('state', 'in', ('to_send', 'to_cancel'))])._process_documents_web_services(job_count=20)</field>
+        <field name="code">model.search([('state', 'in', ('to_send', 'to_cancel'))])._process_documents_web_services(job_count=20, with_commit=True)</field>
```

Comment:
you can't do that here because it forces the user to upgrade `account_edi`. Do it in your new module instead.

Reply:
Ok, but also, what about those who won't install the new module ?

Actually I changed the default value for with_commit. This allows to leave the cron as it is and makes more sense. By default, we should commit.